backend/src/routes/projects/closeout.js

```diff
@@ -1,13 +1,15 @@
 const controller = require("@controllers/projects/closeout");
 const validators = require("@validators/projects/closeout");
 const what = "projects";
+const ROLE = "PMO-Manager-Edit-Capability" || "PMO-Admin-Edit-Capability";
```

Comment:
I'm curious about this, one, as i know this close out is a one off, and requires something different, depending on the user.  Maybe we can chat about this one